lib/Room.js

```diff
@@ -56,6 +56,7 @@ class Room extends EventEmitter {
       audioLevelObserver,
       activeSpeakerObserver,
       consumerReplicas,
+      maxPeersCount,
```

Comment:
```suggestion
      maxPeerCount,
```

Correct spelling for this parameter is `maxPeerCount`

Reply:
Resolved.